pkg/sessions/cookie/session_store.go

```diff
@@ -136,18 +136,18 @@ func splitCookie(c *http.Cookie) []*http.Cookie {
 	valueBytes := []byte(c.Value)
 	count := 0
 	for len(valueBytes) > 0 {
-		new := copyCookie(c)
-		new.Name = fmt.Sprintf("%s_%d", c.Name, count)
+		cookie := copyCookie(c)
```

Comment:
The package this code is in is called `cookie` so we are now shadowing that I believe. Maybe we should use a completely different name here?

Reply:
Now  name is newCookie in https://github.com/pusher/oauth2_proxy/commit/e64e6fa514e8e4e0df14135f1e2269824f075c0b